pytest.ini

```diff
@@ -20,6 +20,7 @@ markers =
     graceful_error: 'graceful_error' tests
     database_manager: database tests
     galaxy: marks a test as a galaxy test
+    ratmama: marks a test as a ratsignal parser test
```

Comment:
I think this mark should be changed to something like 'parser or r**atsignal_parse*

Reply:
But I did say it?

Reply:
@shatteredbeam nitpicked on the NAME, to which I agree, it is not perfect. @theunkn0wn1 "agreed", but nitpicked on the description string, which can be extended to provide way more information, but "a RSignal parser test" should be sufficient to know what the mark is for.